cmd/capabilities.go

```diff
@@ -35,25 +31,43 @@ capabilitiesToBeDropped:
   - SETFCAP #Set file capabilities.
 `
 
+var defaultCapList = &KubeauditConfigCapabilities{
+	// SANE DEFAULTS:
+	NetAdmin:       "drop",
+	SetPCAP:        "drop",
+	MKNOD:          "drop",
+	AuditWrite:     "drop",
+	Chown:          "drop",
+	NetRaw:         "drop",
+	DacOverride:    "drop",
+	FOWNER:         "drop",
+	FSetID:         "drop",
+	Kill:           "drop",
+	SetGID:         "drop",
+	SetUID:         "drop",
+	NetBindService: "drop",
+	SYSChroot:      "drop",
+	SetFCAP:        "drop",
+}
+
 func recommendedCapabilitiesToBeDropped() (dropCapSet CapSet, err error) {
-	yamlFile := []byte(defaultDropCapConfig)
-	if rootConfig.dropCapConfig != "" {
-		if _, err = os.Stat(rootConfig.dropCapConfig); err != nil {
-			return
-		}
-		yamlFile, err = ioutil.ReadFile(rootConfig.dropCapConfig)
+	var kubeauditConfig = &KubeauditConfig{}
+	if rootConfig.kubeauditConfig != "" {
+		data, err := ioutil.ReadFile(rootConfig.kubeauditConfig)
 		if err != nil {
-			return
+			log.Println(err)
+			return dropCapSet, err
 		}
-	}
-	caps := capsDropList{}
-	err = yaml.Unmarshal(yamlFile, &caps)
-	if err != nil {
-		return
-	}
-	dropCapSet = make(CapSet)
-	for _, drop := range caps.Drop {
-		dropCapSet[CapabilityV1(drop)] = true
+
+		yaml.Unmarshal(data, kubeauditConfig)
```

Comment:
does this not return an `err`?

Reply:
It does but effectively this err is unreachable, we check if Unmarshaling is fine in root init, if it isn't kubeaudit crashes anyways

Reply:
maybe document this so that other people don't have the same question as I did when reading the code